test/parser.jl

```diff
@@ -896,6 +896,11 @@ end""" |> test_expr
         end
     end
     @testset "bad uint" begin
-            @test Expr(CSTParser.parse("0x.")) == Expr(:error)
-        end
+        @test Expr(CSTParser.parse("0x.")) == Expr(:error)
+    end
+
+    @testset "endswithtrivia" begin
+        x = CSTParser.parse(raw""""some long title $label1 $label2"  a""")
+        @test CSTParser.lastchildistrivia(x[3])
```

Comment:
Might be good to have a test for `x[3].span < x[3].fullspan` here, if I understand this correctly.

Reply:
You understand correctly - that is what it's testing via `lastischildistrivia`, but I've added another explicit test